marketplace-ui/src/app/modules/product/product-detail/product-detail-feedback/product-feedbacks-panel/product-feedback/product-feedback.component.spec.ts

```diff
@@ -4,11 +4,14 @@ import { CommonModule } from '@angular/common';
 import { StarRatingComponent } from '../../../../../../shared/components/star-rating/star-rating.component';
 import { ElementRef } from '@angular/core';
 import { Feedback } from '../../../../../../shared/models/feedback.model';
+import { MissingTranslationHandler, TranslateLoader, TranslateModule, TranslateService } from '@ngx-translate/core';
+import { httpLoaderFactory } from '../../../../../../core/configs/translate.config';
 
 describe('ProductFeedbackComponent', () => {
   let component: ProductFeedbackComponent;
   let fixture: ComponentFixture<ProductFeedbackComponent>;
   let mockElementRef: ElementRef;
+  let translateService: TranslateService;
```

Comment:
If you remove it, the test of this component will fail because in the UI, we have used the time-ago pipe, this pipe need some libraries that are related to TranslateService.